docs/report/report.tex

```diff
@@ -385,14 +385,60 @@ \subsubsection{Retrospektive}
 \colorbox{red}{TODO:}Was haben wir dabei gelernt? Neue Erkentnisse? Neue Sichtweisen?
 Was lief gut, neu gelernt, was lief nicht so gut, was verbessern?
 
-\section{Erkenntnisse}
-
 \section{Code Walkthrough}
 \colorbox{red}{TODO:}vielleicht interessante Code Snippets?
 
 \section{Testing}
 \colorbox{red}{TODO:}wie haben wir getestet, haben wir getestet?
 
+\section{Fazit}
+\subsection{Probleme, Lösungen, Erkenntnisse}
+\colorbox{red}{TODO: Finde TITEL:} Fazit oder nur Erkenntnisse oder oder?
+\colorbox{red}{Besseres Intro zum Kapitel?}
+Das Projekt Informatik gab einen Einblick in die Arbeitswelt, das Team musste sich klassischen Alltagsproblemen stellen, konnte Lösungswege finden und wichtige Erkenntnisse für die Zukunft daraus ziehen.
+Auch Coding-Entwurfsmuster und -Konventionen wurden neu erlernt und praktisch angewandt.
+
+\subsubsection{Einarbeitung in neue Technologien}
+Zu Beginn des Projektes bestand eine größere Differenz, inwieweit die einzelnen Teammitglieder mit den jeweiligen Technologien vertraut waren.
+Sich mit der Projektstruktur vertraut zu machen war eine größere Hürde.
+Durch die vielen automatisch generierten Dateien und Ordner wirkte die Komplexität und somit Einstiegshürde groß, und Teammitglieder brauchten Ihre Zeit, um effektiv mitarbeiten zu können.
```

Comment:
```suggestion
Insbesondere die Projektstruktur bereitete aufgrund der höheren Komplexität eine größere Einstiegshürde für viele Entwickler. Dies lag darin begründet, dass die vielen automatisch generierten Dateien und Ordner anfangs zu Unübersichtlichkeit führten. Mit zunehmender Entwicklungszeit und durch die Arbeit an den Teilkomponenten des Mono-Repositorys gewannen die Entwickler jedoch schnell mehr Überblick über die Struktur.
```

So ganz zufrieden bin ich auch noch nicht mit meinem Vorschlag...

Reply:
doch find ich gut, hab ich so übernommen :)

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/419/commits/c5dae29b261f99d39aac66c889e6bb96205a2f1a